site/pages/index.tsx

```diff
@@ -5,18 +5,24 @@ import {
   getTreasuryBalance,
 } from "@klimadao/lib/utils";
 import { Home, Props } from "components/pages/Home";
+import { loadTranslation } from "@klimadao/site/lib/i18n";
```

Comment:
This is done.
I had to add the locale to redux to make sure components could hook to locale changes change. Otherwise Localizing the date was not updating at the right time.